compose/snippets/src/main/java/com/example/compose/snippets/LandingScreen/LandingScreen.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.compose.snippets.LandingScreen
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun LandingScreen(
+    toBrushExamples: () -> Unit,
```

Comment:
What do you think about making something like a Destination enum that contains a list of destinations, with their paths, then we can put them in a LazyColumn list of items.

I did this in my playground of experiments and it works pretty well when adding more and more snippets. https://github.com/riggaroo/compose-playtime/blob/main/app/src/main/java/dev/riggaroo/composeplaytime/MainActivity.kt#L56

Reply:
Done. I mimicked your approach and implemented it in this [commit](https://github.com/android/snippets/pull/130).

Reply:
Nice, you can remove the specific callbacks here then, and just have a generic, `onNavigateTo: (Destination) -> Unit` parameter, then you only need one callback.

Reply:
Done. I more thoroughly copied the example you linked earlier.